java-checks/src/test/java/org/sonar/java/checks/UselessPackageInfoCheckTest.java

```diff
@@ -45,7 +45,17 @@ void withOtherFile() {
   @Test
   void notAPackageInfo() {
     JavaCheckVerifier.newVerifier()
-      .onFile(testSourcesPath("checks/UselessPackageInfoCheck/packageWithNoOtherFilesButNotPackageInfo/HelloWorld.java"))
+      .onFiles(
```

Comment:
To avoid  the silent corruption of one of the test files,  we should probably repeat the test with a call to singular `onFile`.